### ISSUE TITLE
Need markup that can account for spatial materials

### ISSUE THREAD

**Issue by wfree-aph** [daisy/ebraille#30]
As a transcriber, sometimes I must present materials in a way that requires multiple parts of the same line to begin in specific cells and for the material to also begin in those cells on subsequent lines.

*Detail:*
Spatial content is a normal part of braille transcription. An example would be vertically arraigned math problems. These kinds of braille materials would require, for example, that each line of the first problem begin in the 3rd cell, each line of the second problem to begin in the 8th cell, and so on. Special markup will be required to ensure that this content can be presented dynamically, rather than requiring a fixed layout.

*Proposal:*
In order to avoid requiring that these kinds of spatial materials have a fixed layout, we need markup that allows for the different parts of the spatial material to be arraigned dynamically. This could require using a container that denotes the beginning and end of the spatial materials, while also using markup specific to each part of the spatial material, so that it can be laid out in a way that respects its spatial nature but while also allowing it to be rearraigned to fit varying line lengths.

**Comment by mrhunsaker** [comment]
This is also important for worksheet layout in Nemeth and UEB Technical for math and science materials. Not to mention poetry and complicated diagramming for college courses in logic. 

I think having a method to do this that designates based on cell would be optimal

**Comment by jrbowden** [comment]
HTML table can define position based on percentages or absolute sizes, is this the kind of thing you mean? e.g. the first olumn of a table could be three cells, the second column could be 50% and the third 30%. Unsure how one could specify this or if mixing absolute and relative sizing is valid.
When you have different line lengths, do you need to specify a minimum and/or maximum size?
I also like to mention the spatial layout of bar-over-bar music - that cannot easily be rearranged but still needs the left hand to line up with the right hand in keyboard music for example.

**Comment by Steve-Noble** [comment]
I agree that this use case makes a lot of sense when there is a need to support dynamic layout of math expressions in a variety of braille displays, where one needs to keep a long expression from line-breaking in such a way that the spatial layout gets messed up when reading it on different size displays.

**Comment by wfree-aph** [comment]
Reopening because of issues raised in #231.
